src/components/video/index.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Video from ".";
+
+<Meta title="Components/Video" component={Video} />
+
+# Video
+
+--- Add a description of the component here ---
+
+## Usage
+
+```jsx
+import { Video } from "@wpmedia/arc-themes-components";
+
+const Feature = () => <Video>Component Children</Video>;
+```
+
+## Properties
+
+<ArgsTable of={Video} />
+
+## Stories
+
+** Video Half Vertical Viewport Loading **
+
+<Canvas>
+	<Story name="Video Half Vertical Viewport Loading">
+		<Video
+			viewportPercentage={50}
+			aspectRatio={1 / 0.562}
```

Comment:
dividing `1 /` because the incoming aspect-ratio on the video is `1 / 0.562` aspect ratio actually even though it's listed as 0.562 in its data- attribute